src/components/calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './calculator.css';
+
+function Calculator() {
+  return (
+    <div className="Container">
```

Comment:
- [ ] It would be nice if you start your calculator component name with a capital `C` instead of a small `c` because of two reasons :arrow_down:

1. It is mentioned in the requirements that you should have a component Calculator.js so in the requirements it started with a capital `C` :blush:
2. It is important to know the naming convention because in React a component name starts with a capital letter and logic files (Not components) starts with small letters :+1: